pkg/config/app_config.go

```diff
@@ -124,21 +124,40 @@ func findOrCreateConfigDir() (string, error) {
 	return folder, os.MkdirAll(folder, 0o755)
 }
 
-func loadUserConfigWithDefaults(configFiles []string) (*UserConfig, error) {
-	return loadUserConfig(configFiles, GetDefaultConfig())
-}
+func loadUserConfig(configFiles []string) (*UserConfig, *AppState, error) {
+	userConfig := GetDefaultConfig()
+	appState := GetDefaultAppState()
+
+	// load recorded AppState from file
+	filepath, err := configFilePath("state.yml")
+	if err != nil {
+		if os.IsPermission(err) {
+			// todo: apparently when people have read-only permissions they prefer us to fail silently
+			return userConfig, appState, nil
```

Comment:
We should still obtain the user config if we fail to load state.yml: right now it's returning the default user config.